### ISSUE TITLE
Issues after getting SD card

### ISSUE THREAD

**Issue by 0fMothsAndMen** [moraroy/NonSteamLaunchers-On-Steam-Deck#302]
I only recently got my deck but my SD card hadn't arrived yet. The initial set-up for both the launchers and scanner had no issues though I only installed one game. My SD arrived and I reconfigured NSL to move my launchers and download games to the SD card. Launchers worked fine at first, game was downloaded to the SD card, but for some reason NSLGameScanner wouldn't detect the game.
Specifically, I am currently testing with a Ubisoft title. I have not touched any settings other than the "move to SD card" feature and downloading the game from the launcher to the default SD card folder. I did not use separate prefixes and have tried uninstalling and reinstalling, manual restarts, stopping and restarting NSLGameScanner in the desktop app, and using both manual and auto scanning with the decky plugin. At one point it kept giving me duplicate Ubisoft Connect and Epic Games shortcuts. 
I am now at a place where I can open the game but must use the launcher to start up and cloud save doesn't work. Once or twice the launcher "lost" the game location on the SD card and wouldn't accept it as already downloaded. I fixed this by manually going into the properties in Steam and restoring the target folder to the one on my SD card. 
Cloud save and NSLGameScanner both worked for this game with no issues when I installed the game to the internal memory. Is there a step or setting I am missing or does Ubisoft Connect specifically not work on the SD card? I noticed it doesn't have the "full SD card support" label here on gh, is that the reason for my problem?

**Comment by moraroy** [comment]
@0fMothsAndMen yes that's a exactly the reason. Ubisoft doesn't have that coding yet for the scanner, there should be no issues if you select your SD Card from "within the launcher" to install your games. NSL has the move to your SD card, for the entire prefix, still working on this to make it seamless for the plugin version.

**Comment by 0fMothsAndMen** [comment]
Thank you so much!! I am brand new to the Steam Deck so I spent quite a long time yesterday thinking I did something wrong, glad to hear it's in the works!
